client/src/screens/EditElectionBallotPackageScreen.tsx

```diff
@@ -19,17 +30,36 @@ const EditElectionBallotPackageScreen = () => {
       }
 
       for (const file of files) {
-        const ballotPackage = await readBallotPackage(file)
+        try {
+          const ballotPackage = await readBallotPackage(file)
+
+          await putElectionDefinition({ electionId, election: ballotPackage.election })
+          const uploadBallot = async ({ ballotStyle, file, precinct }: BallotPackageEntry) => await putBallotTemplate({
+            ballotStyleId: ballotStyle.id,
+            electionId,
+            file,
+            precinctId: precinct.id,
+          })
+          ballotPackage.ballots.forEach(uploadBallot)
```

Comment:
This will start uploading all the PDFs at once, but will ignore errors and won’t show any progress, right?

Reply:
😉 Added!

Reply:
Err… cribbed from your `bsd` code. Still ignores errors.